windows.js

```diff
@@ -129,7 +140,8 @@ async function installMSYS2Tools() {
 // windows-2022 and later images
 export async function installJRubyTools() {
   await installMSYS2Tools()
-  await installGCCTools('mingw64')
+  // Will need to add OpenSSL version detection when JRuby uses OpenSSL 3.0 ?
```

Comment:
I think JRuby never uses libssl, but rather JDK stuff or bouncycastle (I think both not using libssl)